spec/system/annual_reports_system_spec.rb

```diff
@@ -26,7 +26,7 @@
       visit subject
       year = 1.year.ago.year
       click_on(year.to_s)
-      expect(page).to have_content("Diaper Acquisition")
+      expect(page).to have_content("Diapers")
```

Comment:
This test would have passed while the heading was still "Diaper Acquisition".  because there are other cases of "Diapers" in the report.

Reply:
done with your request @cielf